### ISSUE TITLE
Best way to use in Sketch for design?

### ISSUE THREAD

**Issue by ziyafenn** [riccardoscalco/textures#31]
Hey guys, love these textures. Any way to download use this svg library in Sketch app for design? 
Thanks!

**Comment by riccardoscalco** [comment]
No, but I do not know Sketch so I cannot say if there is a way to make this js module compatible with Sketch.

**Comment by hkrish** [comment]
I do this with Illustrator, I don't have Sketch at work, so can't try this.

 - Make/find some texture you would want to copy, 
 - Open the developer console / inspector in your browser
 - Right click on the SVG node / "edit as HTML" (or the equivalent in your browser)
 - Copy the SVG as text and paste into Illustrator (this might also work in Sketch, haven't tried)

The Swatches panel should have all the patterns defined in the <defs></defs> section of the SVG, so you can delete the svg itself and still use the textures.

**Comment by El-Peacock** [comment]
Couldn't figure out how to do the illustrator hack you mentioned. But being able to design with these patterns would be extremely helpful.
